Libraries/LibGUI/GFileSystemModel.cpp

```diff
@@ -31,6 +31,15 @@ struct GFileSystemModel::Node {
         ASSERT_NOT_REACHED();
     }
 
+    void cleanup()
+    {
+        for(auto &child: children) {
```

Comment:
Please use the same formatting as the rest of the codebase:

```suggestion
        for (auto& child : children) {
```

You can use `clang-format` (version 8 or later) to do this automatically. I have it set up so I just press a key combination in my editor to invoke it, for example :)

Reply:
I'll see if I can get this, looks like linux mint only has version 6.